docs/how-to/env.md

```diff
@@ -0,0 +1,8 @@
+# Pass environment variables
+
+To pass in enviroment variables when creating or upgrading an AnyCloud app define the optional `-e`, or `--env-file`, parameter that is a path to a `.env` file
```

Comment:
It would be good to add an extra sentence that the file consists of `KEY=VALUE` pairs on separate lines.

Reply:
Or to link to, say, Bash or Docker documentation on how they should be formatted.

Reply:
yes